evm/vm/base.py

```diff
@@ -234,40 +471,31 @@ def get_state_class(cls):
 
         return cls._state_class
 
-    def get_state(self):
+    def get_state(self, chaindb=None, block_header=None, prev_headers=None):
```

Comment:
No callsites seem to pass a prev_headers argument to this method. Would it make sense to remove it?